config/env/development.example.env

```diff
@@ -0,0 +1,28 @@
+# Application
+SECRET_KEY_BASE: ""
```

Comment:
Why on development env is SECRET_KEY_BASE? and on test is CONFIG_SECRET_KEY_BASE ?